src/data_designer/engine/dataset_builders/artifact_storage.py

```diff
@@ -169,19 +176,21 @@ def write_batch_to_parquet_file(
         batch_number: int,
         dataframe: pd.DataFrame,
         batch_stage: BatchStage,
+        subfolder: str = "",
```

Comment:
Wondering if we can avoid this 🤔

Looks like it is hardcoded to `""` in processor implementation? If we do keep it, can up change to a default of `None`?